pkg/pillar/cmd/vaultmgr/vaultmgr.go

```diff
@@ -182,11 +182,13 @@ func getProtectorID(vaultPath string) ([][]string, error) {
 func changeProtector(vaultPath string) error {
 	protectorID, err := getProtectorID(vaultPath)
 	if protectorID != nil {
-		if err := stageKey(true, oldKeyDir, oldKeyFile); err != nil {
+		//cloudKeyOnlyMode=true, useSealedKey=false
+		if err := stageKey(true, false, oldKeyDir, oldKeyFile); err != nil {
 			return err
 		}
 		defer unstageKey(oldKeyDir, oldKeyFile)
-		if err := stageKey(false, keyDir, keyFile); err != nil {
+		//cloudKeyOnlyMode=false, useSealedKey=false
+		if err := stageKey(false, false, keyDir, keyFile); err != nil {
 			return err
 		}
 		defer unstageKey(keyDir, keyFile)
```

Comment:
I assume this is to get rid of the non-sealed vault key, but shouldn't we immediately setup the sealed one?
What happens if there is a power outage after this (assumed by me) removal and the creation of the sealed one?

Reply:
As mentioned above, this is for deprecated vaults only, so this is not related to the default vault (/persist/vault) and its sealed key.

Reply:
If there is a power outage after the execCmd call - the key would have moved to TPM based key - which is expected
if there is a power outage before the execCmd call - the key would not have moved to TPM based key - which will trigger the post-reboot session to try changeProtector again.

Reply:
Explained the same as a comment in the code